website/docs/reference/resource-configs/databricks-configs.md

```diff
@@ -7,7 +7,7 @@ id: "databricks-configs"
 
 When materializing a model as `table`, you may include several optional configs that are specific to the dbt-databricks plugin, in addition to the standard [model configs](/reference/model-configs).
 
-<VersionBlock lastVersion="1.5">
+<VersionBlock >
```

Comment:
just double-checking, is this whole block supposed to be deleted (ie delete lines 10 - 21)?